snoop/data/language_detection/polyglot.py

```diff
@@ -0,0 +1,11 @@
+from polyglot.detect import Detector
+
+
+NAME='polyglot'
+
+def detect(txt):
+    detector = Detector(txt)
+    if detector.language.code and detector.language.code != 'un' and \
```

Comment:
Please avoid having to split long lines with `\`. You can extract `detector.language.code` into a variable and make the whole function more readable.